reactions/src/HALO.cpp

```diff
@@ -1188,7 +1194,7 @@ double HALO::PHALO_pseudo(double k, bool mgcamb) const{
     mypkl = pow2(linear_growth(k))*P_l(k);
   }
 
-  if (phpars_pseudo[0] == 1000. || k<=0.005) {
+  if (phpars_pseudo[0] == 1000. || k<=0.01) {
```

Comment:
Sorry I'm really nitpicking now, but what does this line do? And should 0.01 be a named variable e.g. k_max or something a bit more descriptive?

Reply:
Ah this is some selection criteria to transition from completely linear to non-linear for halofit. Details would be in Takashi et al 1208.2701. I don't think these need to be commented on so much, I have included the Takashi et al reference at the top of the formula for details.

Reply:
Ah ok, thanks.